modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/Table/index.js

```diff
@@ -166,6 +166,14 @@ const Table = ({
 										expanded={column.expanded}
 										key={`${rowIndex}-${columnIndex}`}
 										noWrap={column.noWrap}
+										onClick={() => {
+											if (
+												!column.disableCustomClickOnRow &&
+												row.customClickOnRow
+											) {
+												return row.customClickOnRow();
+											}
+										}}
```

Comment:
Why this function is inside of `onClick` and not on scope file?

Reply:
It is just a verification and a callback to click on row

Reply:
Because this is a validation and a callback to a custom click on Table row. So this will only create more complexity